libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -538,17 +538,25 @@ def create_cache_point(cls, cache_type: str = "default") -> Dict[str, Any]:
     @classmethod
     def build_extra(cls, values: dict[str, Any]) -> Any:
         """Build extra kwargs from additional params that were passed in."""
-        all_required_field_names = get_pydantic_field_names(cls)
-        values = _build_model_kwargs(values, all_required_field_names)
-
-        # Merge model_kwargs (name assumed in langchain-core) and
-        # additional_model_request_fields (name used in ChatBedrockConverse)
         model_kwargs = values.pop("model_kwargs", {})
         additional_model_request_fields = values.pop(
             "additional_model_request_fields", {}
         )
-        if additional_model_request_fields or model_kwargs:
+        if model_kwargs:
+            model_kwargs_msg = (
+                "Please use additional_model_request_fields instead of "
+                "model_kwargs for any extra inference parameters."
+            )
+            logger.warning(model_kwargs_msg)
+            warnings.warn(model_kwargs_msg)
```

Comment:
Are these 2 redundant? Also, a form like this will help folks filter these.

```python
if model_kwargs:
        warnings.warn(
            "ChatBedrockConverse uses 'additional_model_request_fields' instead of 'model_kwargs'. "
            "Your parameters have been automatically converted.",
            UserWarning,
            stacklevel=2
        )

```

Reply:
Thanks 👍 Removed logger statement and updated with suggested params